Content.Shared/Xenoarchaeology/Equipment/Components/ArtifactAnalyzerComponent.cs

```diff
@@ -16,6 +16,21 @@ public sealed partial class ArtifactAnalyzerComponent : Component
     [DataField]
     public TimeSpan AnalysisDuration = TimeSpan.FromSeconds(30);
 
+    // DeltaV - Summary - Begin modified code block: tie artifacts to glimmer.
+    /// <summary>
+    /// The ratio of research points per one glimmer.
+    /// </summary>
+    public int ExtractRatio = 750;
+    // Nyano - End modified code block.
+
+    // Begin DeltaV - Tie point output to glimmer
+    /// <summary>
+    /// The maximum added multiplier, reached at max glimmer.
+    /// </summary>
+    [DataField]
+    public float PointGlimmerMultiplier = 8f;
+    // End DeltaV - Tie point output to glimmer
+
```

Comment:
probably simpler to wrap both statements in DeltaV comments like this. also missed a nyano reference :trollface: 
```suggestion
    // Begin DeltaV - Tie point output to glimmer
    /// <summary>
    /// The ratio of research points per one glimmer.
    /// </summary>
    public int ExtractRatio = 750;

    /// <summary>
    /// The maximum added multiplier, reached at max glimmer.
    /// </summary>
    [DataField]
    public float PointGlimmerMultiplier = 8f;
    // End DeltaV - Tie point output to glimmer
```